source/_integrations/govee_light_local.markdown

```diff
@@ -22,4 +22,4 @@ To enable local control on your Govee device, refer to the instructions availabl
 
 ## Supported devices
 
-H619Z, H6072, H619C, H7060, H619B, H6066, H619D, H619E, H61A1, H61A3, H61A2, H618A, H619A, H61A0, H6117, H6159, H6163, H6056, H6076, H6062, H6061, H6046, H6047, H6065, H6087, H610A, H610B, H6172, H61B2, H61E1, H7061, H7062, H7065, H6067, H6059, H6051, H6073, H618F, H7050, H61A5, H618C, H618E, H61A8, H7055, H6078, H705A, H6168, H705B, H7041, H7042, H615E, H7051, H7028, H6173, H7013, H7012, H7021, H70C1
+H619Z, H6072, H619C, H7060, H619B, H6066, H619D, H619E, H61A1, H61A3, H61A2, H618A, H619A, H61A0, H6117, H6159, H6163, H6056, H6076, H6062, H6061, H6046, H6047, H6065, H6087, H610A, H610B, H6172, H61B2, H61E1, H7061, H7062, H7065, H6067, H6059, H6051, H6073, H618F, H7050, H61A5, H618C, H618E, H61A8, H7055, H6078, H705A, H6168, H705B, H7041, H7042, H615E, H7051, H7028, H6173, H7013, H7012, H7021, H70C1, H6052, H6088, H608A
```

Comment:
PR https://github.com/home-assistant/home-assistant.io/pull/31454 (against `current`) just sorted this list, so we should keep that:
```suggestion
H6046,
H6047,
H6051,
H6052,
H6056,
H6059,
H6061,
H6062,
H6065,
H6066,
H6067,
H6072,
H6073,
H6076,
H6078,
H6087,
H6088,
H608A,
H610A,
H610B,
H6117,
H6159,
H615E,
H6163,
H6168,
H6172,
H6173,
H618A,
H618C,
H618E,
H618F,
H619A,
H619B,
H619C,
H619D,
H619E,
H619Z,
H61A0,
H61A1,
H61A2,
H61A3,
H61A5,
H61A8,
H61B2,
H61E1,
H7012,
H7013,
H7021,
H7028,
H7041,
H7042,
H7050,
H7051,
H7055,
H705A,
H705B,
H7060,
H7061,
H7062,
H7065,
H70C1
```